command/helpers.go

```diff
@@ -542,3 +542,14 @@ func (w *uiErrorWriter) Close() error {
 	}
 	return nil
 }
+
+// stringsContains returns true if s is present in the vs string slice
+func stringsContain(vs []string, s string) bool {
```

Comment:
There's a [`helper.SliceStringContains`](https://github.com/hashicorp/nomad/blob/v1.0.4/helper/funcs.go#L190-L198) that implements exactly this function. (It's perhaps inevitable in a code base this size that we have multiple implementations of it floating around 😁 )